### DISCUSSION TITLE
better Spelling / Grammar / Style Check through Language Tool integration

### DISCUSSION THREAD

**Discussion by onemenzel** [Zettlr/Zettlr#3497] [Ideas]
While spelling correction and grammar / style suggestions are an important feature in academic writing, the current spelling correction is severely limited. One solution to improve this situation would be to integrate support for [Language Tool](https://languagetool.org/), which besides its online service also has the privacy friendly option to run the service on ones own machine.

It would really improve my writing process if Zettlr integrated this. Right now, if I want to have this level of writing suggestions I have to copy my whole work into another text editor like Word or some online text box. I would hope that my writing experience could be improved on that end.

The [obsidian language tool integration plugin](https://github.com/Clemens-E/obsidian-languagetool-plugin) gives an idea on how to implement this using Code Mirror.

**Comment by nathanlesage** [comment]
There is an issue out there already for this, and I have plans to implement that — as soon as I find the time.

**Reply by onemenzel** [reply]
Nice, thanks!
